src/Translatable/Contracts/Translatable.php

```diff
@@ -14,6 +14,12 @@ public static function disableAutoloadTranslations(): void;
 
     public static function enableAutoloadTranslations(): void;
 
+    public static function defaultDeleteTranslationsCascade(): void;
```

Comment:
I would remove this because it's equal to the disable one and there is no other parameter that controls the behavior in opposite to the autoloading.